tests/manual/tickets/1609/1.js

```diff
@@ -0,0 +1,139 @@
+/* globals document, console, window */
+
+import Plugin from '@ckeditor/ckeditor5-core/src/plugin';
+import Widget from '@ckeditor/ckeditor5-widget/src/widget';
+import Command from '@ckeditor/ckeditor5-core/src/command';
+import ModelRange from '../../../../src/model/range';
+import { toWidget, toWidgetEditable } from '@ckeditor/ckeditor5-widget/src/utils';
+import ButtonView from '@ckeditor/ckeditor5-ui/src/button/buttonview';
+import ClassicEditor from '@ckeditor/ckeditor5-editor-classic/src/classiceditor';
+import Paragraph from '@ckeditor/ckeditor5-paragraph/src/paragraph';
+import Essentials from '@ckeditor/ckeditor5-essentials/src/essentials';
+
+const OUTER_MODEL = 'speaker';
+const INNER_MODEL = 'special-char';
+
+class SpeakerCommand extends Command {
+	execute( { speaker } ) {
+		const model = this.editor.model;
+		const document = model.document;
+
+		model.change( writer => {
+			const blocks = Array.from( document.selection.getSelectedBlocks() );
+			// NB: This is now private API, but a public equivalent doesn't currently exist
+			const range = ModelRange._createFromRanges( blocks.map( block => writer.createRangeOn( block ) ) );
```

Comment:
@scofalik @pjasiun I remember you could not agree whether this method is needed in the public API.

Reply:
I do not remember any reason to keep it protected.

Reply:
We don't use it outside engine, so that's why we thought it can be made protected: https://github.com/ckeditor/ckeditor5-engine/issues/1555

Also, there are issues with its name and return value: https://github.com/ckeditor/ckeditor5-engine/issues/843

I guess we should first resolve https://github.com/ckeditor/ckeditor5-engine/issues/843 before bringing it back to the public API.